CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the "architect-yml" extension will be documented in this file.
 
+## [1.1.0]
+
+- Remove most unnecessary boilerplate features and functionality
+
```

Comment:
Just noticed this but since we haven't released anything yet I don't think we need to be adding anything specific here or changing version numbers yet... we can just have this launch with 1.0.0 "initial release"

Reply:
good point, I can remove it

Reply:
Removed extra version